osscameroon/yotas
# feat(ui): Home page Integration for medium to big screen devices

#### Integration of home page

**Steps to verify:**
Check netlify preview
https://deploy-preview-78--infallible-rosalind-15c59f.netlify.app/
